pandas/io/formats/html.py

```diff
@@ -85,8 +85,13 @@ def write(self, s, indent=0):
         rs = pprint_thing(s)
         self.elements.append(' ' * indent + rs)
 
-    def write_th(self, s, indent=0, tags=None):
-        if self.fmt.col_space is not None and self.fmt.col_space > 0:
+    def write_th(self, s, header=False, indent=0, tags=None):
+        # header=False is for use of this function inside <tbody>
```

Comment:
can you add a proper doc-string here

Reply:
Added a more verbose doc-string.  Didn't use the sphinx formatting style as this isn't a function for public use, but I can change it to that if you'd like.